salt/orch/kubernetes.sls

```diff
@@ -19,12 +19,20 @@ update_pillar:
     - require:
       - salt: hostname_setup
 
+update_grains:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_grains
```

Comment:
Thanks. This is what I had before @grahamhayes told me `refresh_grains` was not available in our Salt.